src/backend/distributed/executor/multi_utility.c

```diff
@@ -676,7 +676,6 @@ static Node *
 ProcessAlterObjectSchemaStmt(AlterObjectSchemaStmt *alterObjectSchemaStmt,
 							 const char *alterObjectSchemaCommand, bool isTopLevel)
 {
-
```

Comment:
Yeah, `citus_indent` did it; I suppose I can move it to another commit.

I'm inching ever closer to having Travis builds fail if `citus_indent` fails.